src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -338,7 +338,24 @@ public function should_load( $query = null ) {
 	 * @return string Path to the File that initializes the template
 	 */
 	public function filter_template_include( $template ) {
-		$query = tribe_get_global_query_object();
+		$query   = tribe_get_global_query_object();
+		$context = tribe_context();
+
+		/**
+		 * Allows filtering the loading of our proprietary templates.
+		 *
+		 * @since TBD
+		 *
+		 * @param boolean        $load     Whether we should load the theme templates instead of the Tribe templates. Default false.
+		 * @param string         $template The template located by WordPress.
+		 * @param Tribe__Context $context  The singleton, immutable, global object instance.
+		 */
+		$load_template = apply_filters( 'tribe_events_views_v2_should_load_default_templates', false, $template, $context, $query );
```

Comment:
I think the name could be made clearer: "default" does not come with a context to which that "default" should apply.

I would go with `tribe_events_views_v2_use_wp_template_hierarchy`; I think this leaves little to the imagination.